README.md

```diff
@@ -1,307 +1,29 @@
-# Elastic Enterprise Search connectors
+# Elastic connectors
 
-![logo](logo-enterprise-search.png)
-
-The home of Elastic Enterprise Connector Clients. This repository contains the framework for customizing Elastic Enterprise Search native connectors, or writing your own connectors for advanced use cases.
+The home of Elastic connector service and native connectors. This repository contains the framework for customizing Elastic native connectors, or writing your own connectors for advanced use cases.
 
 **The connector will be operated by an administrative user from within Kibana.**
 
 > Note: The connector framework is a tech preview feature. Tech preview features are subject to change and are not covered by the support SLA of general release (GA) features. Elastic plans to promote this feature to GA in a future release.
 
 Before getting started, review important information about this feature:
 
-- [Terminology](#terminology)
-- [Getting help and providing feedback](#getting-help-and-providing-feedback)
+- [Terminology](docs/SUPPORT.md)
+- [Getting help and providing feedback](docs/SUPPORT.md)
+- [Understand the connector protocol](link to elastic docs)
```

Comment:
Hey @wangch079 . I'm not sure of the timeline to move that content into the Elastic docs, since I'm behind on native connector docs. I suggest we continue to link to the connector protocol document within the repository for now.

Later, we can create a new page in the Elastic docs and update any links and documents in the Ruby and Python repositories.

Reply:
Noted, I will work on the connector protocol and let both Ruby and Python link to it. Thanks

Reply:
Perfect, thank you!